hardware_interface/src/joint_state_handle.cpp

```diff
@@ -21,18 +21,13 @@
 
 namespace hardware_interface
 {
-
 JointStateHandle::JointStateHandle()
-: name_(),
-  pos_(nullptr),
-  vel_(nullptr),
-  eff_(nullptr)
-{}
+: name_(), pos_(nullptr), vel_(nullptr), eff_(nullptr)
+{
+}
 
 JointStateHandle::JointStateHandle(
-  const std::string & name,
-  const double * pos,
-  const double * vel,
+  const std::string & name, const double * pos, const double * vel,
```

Comment:
either line break after all, or no line break at all. This one looks weird.

Reply:
all uncrustify, I'll revert the unnecessary style changes